tests/Dialect/DIP/morph2D_f32.mlir

```diff
@@ -0,0 +1,114 @@
+//
+// x86
+//
+// RUN: buddy-opt %s -lower-dip="DIP-strip-mining=64" -arith-expand --convert-vector-to-scf --lower-affine --convert-scf-to-cf --convert-vector-to-llvm \
+// RUN: --convert-memref-to-llvm --convert-func-to-llvm --reconcile-unrealized-casts  \
+// RUN: | mlir-cpu-runner -O0 -e main -entry-point-result=i32 \
+// RUN: -shared-libs=%mlir_runner_utils_dir/libmlir_runner_utils%shlibext,%mlir_runner_utils_dir/libmlir_c_runner_utils%shlibext \
+// RUN: | FileCheck %s
+
+memref.global "private" @global_input : memref<3x3xf32> = dense<[[0. , 1. , 2. ],
+                                                                 [10., 11., 12.],
+                                                                 [20., 21., 22.]]>
+
+memref.global "private" @global_identity : memref<3x3xf32> = dense<[[0., 0., 0.],
+                                                                    [0., 1., 0.],
+                                                                    [0., 0., 0.]]>
+
+memref.global "private" @global_outputerosion : memref<3x3xf32> = dense<[[0., 0., 0.],
```

Comment:
Many of these `memrefs` in this file have same contents, can you specify them with a single variable to reduce code duplication?

On a separate note, although tests involving `OpenCV` in example file cover a wide variety of cases, I would suggest you to add more complicated cases (their size could be 3x3, that's fine by me) on pure `MLIR/LLVM` side of testing as well. 

Please consider this suggestion for all other test files as well.